runtime/lib/query/Criteria.php

```diff
@@ -1239,8 +1239,7 @@ public function isSingleRecord()
      */
     public function setLimit($limit)
     {
-        // TODO: do we enforce int here? 32bit issue if we do
-        $this->limit = $limit;
+        $this->limit = (int) $limit;
```

Comment:
_Maintainers:_
Can anyone provide some context around this comment? My unit tests show that no truncation occurs for large values like 64-bit integers.